_source/_posts/2024-04-08-third-party-cookies-okta-sign-in-widget.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog_post
+title: "How to prepare your self-hosted Okta Sign-In Widget to work without third-party cookies"
+author: edunham
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "If you have a self-hosted Okta Sign-In Widget, there are several options to mitigate Chrome's third-party cookies deprecation."
+tags: [third-party-cookies, okta-sign-in-widget]
+tweets:
+- ""
+- ""
+- ""
+image: blog/3pc/social.jpg
```

Comment:
blog image is the same as the 2/29 post?

Reply:
yeah thanks to Security(TM) I don't actually have access to appropriate software for making a new one right now :/